pkg/cloudprovider/cloudprovider.go

```diff
@@ -65,7 +61,7 @@ var _ cloudprovider.CloudProvider = (*CloudProvider)(nil)
 
 type CloudProvider struct {
```

Comment:
nit: Can we order things in the struct in the same order that they are in the return statement on L70?